pubsub/streaming-analytics/README.md

```diff
@@ -118,6 +119,18 @@ mvn compile exec:java \
     --windowSize=2"
 ```
 
+Gradle:
```

Comment:
Nit: can we move Gradle before Maven? I think it's a good idea to recommend Gradle over Maven.